yt_dlp/extractor/vvvvid.py

```diff
@@ -14,29 +15,28 @@ class VVVVIDIE(InfoExtractor):
     _VALID_URL = r'%s(?P<show_id>\d+)/[^/]+/(?P<season_id>\d+)/(?P<id>[0-9]+)' % _VALID_URL_BASE
     _TESTS = [{
         # video_type == 'video/vvvvid'
-        'url': 'https://www.vvvvid.it/#!show/434/perche-dovrei-guardarlo-di-dario-moccia/437/489048/ping-pong',
-        'md5': 'b8d3cecc2e981adc3835adf07f6df91b',
+        'url': 'https://www.vvvvid.it/show/498/the-power-of-computing/518/505692/playstation-vr-cambiera-il-nostro-modo-di-giocare',
```

Comment:
do the new test URLs match their comment notes?

Reply:
Yes I searched for videos with the same type, and skipped the video/rcs because I found none working in the website, they all throw an error